tool/deploy.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
+
+import 'package:grinder/grinder.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+main(List args) => grind(args);
+
+@DefaultTask('Deploy software')
+deploy() {
+  deployIt();
+}
+
+deployIt() {
+  // TODO: command line parsing (major, minor, patch) version update
```

Comment:
`grinder` doesn't yet support passing args to tasks, but this is under active development.